packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.service.js

```diff
@@ -31,4 +45,42 @@ export default class {
         ),
     );
   }
+
+  filterOwnServiceTypes(serviceTypes) {
+    return this.$http
```

Comment:
Why not prefer using the following module `ngOvhApiWrappers` which is already defined as a peerDependency of the `@ovh-ux/manager-support` module?

See the documentation: https://github.com/ovh-ux/ng-ovh-api-wrappers#paginate

Reply:
As discussed, both solution are ok so we can keep this one since it already received a label "quality check: ok".